building/build_job_cascade_and_where_to_find_build_results.adoc

```diff
@@ -0,0 +1,20 @@
+= Build cascade & results
+
+--
+
+WARNING: This document is old and needs to be updated.
+
+--
+
+Each year our process for building the component projects in JBoss Tools evolves and improves, 
```

Comment:
the basics are the same though.
JBoss Tools -> component builds, main updatesite aggregates, done.

Devstudio -> devstudio component builds, main updatesite aggregates from jbosstools + devstudio components.